test/system/assignments_test.rb

```diff
@@ -28,7 +28,10 @@ class AssignmentsTest < ApplicationSystemTestCase
   test 'assign unassigned client - client side' do
```

Comment:
This whole test is not needed anymore, since there shouldn't be any matching going from the client on.
Which also means the buttons `Freiwillige finden` and `show Assignment` shouldn't be needed anymore. But to be sure I will have more clarification on that today after the meeting with her.